examples/src/index.html

```diff
@@ -37,7 +37,9 @@
 		<div class="footer">
 			Copyright &copy; Jed Watson 2014. MIT Licensed.
 		</div>
-	</div>
+	</div>	
+	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.1/es5-shim.js"></script>
```

Comment:
Without the shim, the example wouldn't run under IE8, so I couldn't verify wether the fix worked.